.github/workflows/ci.yml

```
@@ -12,11 +12,16 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
+    env:
+      BUILD_TYPE: ${{ matrix.build_types }}
+
     strategy:
+      fail-fast: false
       matrix:
         ruby:
           - 2.6
           - 2.7
+        build_types: ["LINT", "REDIS", "ACTIVERECORD"]
```

We could run linting only on the latest ruby:

```suggestion
        build_types: ["LINT", "REDIS", "ACTIVERECORD"]
        exclude:
          - ruby: 2.6
            build_types: "LINT"
```